packages/gatsby-image/gatsby-browser.js

```diff
@@ -0,0 +1,14 @@
+"use strict"
+
+exports.onClientEntry = () => {
+  if (process.env.GATSBY_IMAGE_LOAD_POLYFILLS) {
+    // Add object-fit/position polyfill if browser needs it
+    const testImg = document.createElement(`img`)
+    if (
+      typeof testImg.style.objectFit === `undefined` ||
+      typeof testImg.style.objectPosition === `undefined`
+    ) {
+      import(`object-fit-images`).then(ObjectFitImages => ObjectFitImages())
```

Comment:
This line results in an error for me. I'm not sure the promise understands to use `default` for objects resolved from modules. The following should work:

```
import('object-fit-images').then(({ default: ObjectFitImages }) =>
    ObjectFitImages()
);
```